addon/i3dio/ui/object.py

```diff
@@ -858,12 +887,14 @@ def register():
     bpy.types.Object.i3d_attributes = PointerProperty(type=I3DNodeObjectAttributes)
     bpy.types.Object.i3d_merge_group_index = IntProperty(default = -1)
     bpy.types.Object.i3d_mapping = PointerProperty(type=I3DMappingData)
+    bpy.types.Object.i3d_reference_path = PointerProperty(type=I3DReferencePath)

```

Comment:
Why are you doing this as a PointerProperty to an I3DReferencePath property group with only a string parameter instead of just putting the string here directly?

Reply:
Because I didn't think 😁

Reply:
> Because I didn't think 😁

I figured 😂